docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -50,10 +55,6 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - Features include enabling transfer of rewards and voting rights
 - Items still being discussed:
   - Governance permissioned CosmWASM instance on the hub
-  - Cosmos <> Ethereum Gravity Bridge
-    - Transfer ATOM, ETH, ERC-20, and tokens on the Cosmos Hub between Ethereum- and Cosmos-compatible chains  
-    - Fee and reward model hosted across Cosmos and Ethereum
-    - Adds light-weight infrastructure and operational requirements with minimal slashing conditions to all Hub validators
   - Budget Module
     - Inflation funding directed to arbitrary module and account addresses
   - Farming Module
```

Comment:
I think we can remove farming but may want to double check.

Reply:
would like to wait until there's possible public confirmation.